packages/neuron-ui/src/containers/Navbar/index.tsx

```diff
@@ -95,13 +96,16 @@ const Navbar = () => {
 
   const [verifyCkbResult, setVerifyCkbResult] = useState<VerifyExternalCkbNodeRes>()
 
+  const network = useMemo(() => networks.find(n => n.id === networkID), [networkID, networks])
   useEffect(() => {
-    verifyExternalCkbNode().then(res => {
-      if (isSuccessResponse(res) && res.result) {
-        setVerifyCkbResult(res.result)
-      }
-    })
-  }, [])
+    if (!network?.readonly) {
+      verifyExternalCkbNode().then(res => {
```

Comment:
`Verify external ckb node` should be done in the background if `connected to an unexpected node` should be alerted to users so they can realize it even Neuron is running in the background. We can do this later.